python_scripts/01_tabular_data_exploration_ex_01.py

```diff
@@ -0,0 +1,62 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.6.0
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # 📝 Exercise 01
+
+# %% [markdown]
+# We will try to predict penguins species based on two of their body
+# measurements: culmen length and culmen depth.
+#
+# What are the features? What is the target?
+
+# %% [markdown]
+# The data is located in `../datasets/penguins_classification.csv`, load it
+# with `pandas` into a `DataFrame`.
+
+# %%
+# Write your code here.
+
+# %% [markdown]
+# Show a few samples of the data
+#
+# How many features are numerical? How many features are categorical?
+
+# %%
+# Write your code here.
+
+
+# %% [markdown]
+# How many samples of each species do you have?
```

Comment:
Maybe we can hint to use the `value_counts()` method on a the column of interest of the pandas dataframe.